### DISCUSSION TITLE
README suggestion

### DISCUSSION THREAD

**Discussion by fakecan** [libvips/libvips#4100] [General]
I found out about the Dockerfile guidelines repository (https://github.com/jcupitt/docker-builds/blob/master/libvips-ubuntu22.04/Dockerfile#L18) from this issue (https://github.com/libvips/libvips/issues/3684).
Thanks to this, I completed my long docker setup. If this repository is mentioned in the README, there will be fewer idiots like me.
Also, for no apparent reason, the installation with libvips ubuntu 22.04 from the above repository seems to be slower than the installation with pyvips alone (pyvips is 2.2.3 in both).
Loading an svs file of about 1.2GB is about the same speed, but when .numpy() the object in pyvips.Image.new_from_file(), it is slower by a few seconds.
Of course, this comparison could be wrong.

In any case, I feel comfortable with your library.

**Comment by jcupitt** [comment]
Hello @fakecan,

Do you mean the libdicom install? I think that's documented in the openslide build notes. In any case, when distros switch from openslide3 to openslide4 this problem will go away, so I think we probably shouldn't document it too.

What two libvipses are you comparing? Is your docker build slower than the official ubuntu one? You'd need to share your dockerfile before we could investigate.
